srcpkgs/qt5-webkit/template

```diff
@@ -1,59 +1,40 @@
 # Template file for 'qt5-webkit'
 pkgname=qt5-webkit
-version=5.9.0
-revision=5
-wrksrc=qtwebkit-opensource-src-${version}
+version=5.212.0alpha2
```

Comment:
I think using this as version will prevent us from updating to `5.212.0` release because `5.212.0 < 5.212.0alpha2`.
Perhaps the `alpha2` part should be omitted and appended only in the `distfile` and `wrksrc`.
Then we can just revbump once the `5.212.0` release is available.

Reply:
yeah, that sounds good to me

Reply:
Just tested this, xbps handles `alpha` in version, upgrading from `5.212.0alpha2_1` to `5.212.0_1` works

Reply:
oh, i just updated it; should i revert back to 5.212alpha2?